libs/websoc-api-next/index.ts

```diff
@@ -239,9 +239,12 @@ export const callWebSocAPI = async (
   json.schools.forEach((school) =>
     school.departments.forEach((department) =>
       department.courses.forEach((course) =>
-        course.sections.forEach(
-          (section) => (section.meetings = getUniqueMeetings(section.meetings))
-        )
+        course.sections.forEach((section) => {
+          section.meetings.forEach(
+            (meeting) => (meeting.bldg = [meeting.bldg] as unknown as string[])
```

Comment:
```suggestion
            (meeting) => {
              meeting.bldg = [meeting.bldg].flat()
            }
```
I think this accomplishes the same thing, but without asserting